flake-modules/dev/devshell.nix

```diff
@@ -32,11 +32,30 @@
             {
               name = "tests";
               help = "Run nixvim tests";
-              command = ''
-                echo "=> Running nixvim tests for the '${system}' architecture..."
-
-                ${nix} build .#checks.${system}.tests "$@"
-              '';
+              command =
+                let
+                  launchTest =
+                    pkgs.runCommandLocal "launch-test.sh"
+                      {
+                        script = ./launch-test.sh;
+                        nativeBuildInputs = [ pkgs.makeWrapper ];
+                      }
+                      ''
+                        makeWrapper $script $out/bin/launch-test.sh \
+                          --prefix PATH : ${
+                            lib.makeBinPath (
+                              with pkgs;
+                              [
+                                jq
+                                fzf
+                              ]
+                            )
+                          }
+                      '';
+                in
+                ''
+                  NIXVIM_SYSTEM=${system} NIXVIM_NIX_COMMAND=${nix} ${launchTest}/bin/launch-test.sh "$@"
```

Comment:
```suggestion
                  NIXVIM_SYSTEM=${system} NIXVIM_NIX_COMMAND=${nix} ${lib.getExe launchTest} "$@"
```

Reply:
I need the mkShellApplication for that, but yes!